apps/navstar_overlay/src/navstar_overlay_helper.erl

```diff
@@ -0,0 +1,71 @@
+%%%-------------------------------------------------------------------
+%% @doc navstar public API
+%% @end
+%%%-------------------------------------------------------------------
+-module(navstar_overlay_helper).
+
+%% Application callbacks
+-export([run_command/1, run_command/2]).
+
+-ifdef(TEST).
+-include_lib("eunit/include/eunit.hrl").
+-endif.
+
+run_command(Command, Opts) ->
+    Cmd = lists:flatten(io_lib:format(Command, Opts)),
+    run_command(Cmd).
+
+-spec(run_command(Command :: string()) ->
+    {ok, Output :: string()} | {error, ErrorCode :: non_neg_integer(), ErrorString :: string()}).
+-ifdef(DEV).
+run_command("ip link show dev vtep1024") ->
```

Comment:
What do you think about converting these to netlink calls in a follow-up patch? I think it'd be valuable to talk about netlink, and the kernel interface next week?